src/item/Banner.php

```diff
@@ -90,6 +93,16 @@ public function setPatterns(array $patterns) : self{
 		return $this;
 	}
 
+	public function isIllagerPattern() : bool{
+		return $this->illagerPattern;
+	}
+
+	public function setIllagerPattern(bool $illagerPattern) : self{
+		$this->illagerPattern = $illagerPattern;
+		$this->color = DyeColor::WHITE(); // meta 15 matches vanilla
```

Comment:
This is unexpected behaviour, particularly since the colour could later be changed back to something else.